app/src/main/java/org/mozilla/fenix/settings/creditcards/String.kt

```diff
@@ -17,6 +19,13 @@ fun String.toCreditCardNumber(): String {
     return this.filter { it.isDigit() }
 }
 
+/**
+ * Returns last for digits from a formatted credit card number string
+ */
+fun String.last4Digits(): String {
```

Comment:
The way we use it right now is just in places where `toCreditCardNumber` is already called on the user input, so it would be redundant to call it again.